internal/restorer/filerestorer.go

```diff
@@ -50,10 +52,13 @@ type fileRestorer struct {
 	idx        func(restic.BlobHandle) []restic.PackedBlob
 	packLoader repository.BackendLoadFn
 
-	workerCount int
-	filesWriter *filesWriter
-	zeroChunk   restic.ID
-	sparse      bool
+	workerCount       int
+	filesWriter       *filesWriter
+	zeroChunk         restic.ID
+	sparse            bool
+	progressFormatter *progressformatter.RestoreProgressFormatter
+	terminal          *termstatus.Terminal
+	printProgress     bool
```

Comment:
With the comments from above, both `terminal` and `printProgress` should no longer be necessary.